Gruntfile.js

```diff
@@ -212,6 +255,38 @@ module.exports = function(grunt) {
 
 	});
 
+	grunt.registerTask( 'kebabToCamelCase', 'Rename any svg attributes to camel case for react', function() {
+		var svgFiles = grunt.file.expand( { filter: 'isFile', cwd: 'svg-min/' }, [ '**/*.svg' ] );
+
+		// Add stuff
+		svgFiles.forEach( function( svgFile ) {
+
+			// Grab the relevant bits from the file contents
+			var fileContent = grunt.file.read( 'svg-min/' + svgFile );
```

Comment:
>This was why I moved out the task, and ordered it after the php plugin generation.

Yeah, I guess my point is that `svg-min` aren't really usable SVG files because their props are already camel-cased by this task